percona/controllers/pgc/schedule.go

```diff
@@ -0,0 +1,73 @@
+package pgc
+
+import (
+	"context"
+	"encoding/json"
+	"reflect"
+
+	"github.com/percona/percona-postgresql-operator/cmd/pgo-scheduler/scheduler"
+	crv1 "github.com/percona/percona-postgresql-operator/pkg/apis/crunchydata.com/v1"
+	"github.com/pkg/errors"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (c *Controller) handleScheduleBackup(newCluster, oldCluster *crv1.PerconaPGCluster) error {
+	ctx := context.TODO()
+	if oldCluster != nil && !reflect.DeepEqual(newCluster.Spec.Backup.Schedule, oldCluster.Spec.Backup.Schedule) {
+		for _, schedule := range oldCluster.Spec.Backup.Schedule {
+			cmName := newCluster.Name + "-" + schedule.Name
+			err := c.Client.CoreV1().ConfigMaps(newCluster.Namespace).Delete(ctx, cmName, metav1.DeleteOptions{})
+			if err != nil {
+				return errors.Wrapf(err, "delete config map %s", cmName)
+			}
+		}
+	}
+	if newCluster == nil {
```

Comment:
should we check it before the `DeepEqual` line?

Reply:
yep) https://github.com/percona/percona-postgresql-operator/pull/114/commits/01211f93365b4ea2573729b918536c8536d57abf